client/src/parsec/workspace.ts

```diff
@@ -249,6 +249,27 @@ export async function mountWorkspace(
   return await libparsec.workspaceMount(workspaceHandle);
 }
 
+export async function unmountWorkspace(workspace: WorkspaceInfo): Promise<Result<null, MountpointUnmountError>> {
+  let lastError: MountpointUnmountError | null = null;
+
+  if (workspace.mountpoints.length === 0) {
```

Comment:
Useless. if the length if 0, we're not entering the loop anyway, lastError is not set, we're going straight to line 270 that does exactly the same thing.